### DISCUSSION TITLE
posts/003-write-blog-everywhere/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#3] [Announcements]
# posts/003-write-blog-everywhere/

隨時隨地寫博客就像隨時隨地加班，我不要啊！！！

**Comment by PermeationLoop** [comment]
隨時隨地工作是牛馬，但是隨時隨地黑屁是好文明；

（評論精選已開啓，三十天內已拉黑 114514 條黑屁😁👍🏻請謹言慎行）

**Comment by PermeationLoop** [comment]
小太陽更新： 勾線中．．．．．
俺畫的睫毛真是越來越辣了

<img src="https://github.com/user-attachments/assets/8c3c9c4b-3fda-45f8-a3c6-f229ad3797ad" width="2560" height="1497" alt="小太陽草稿"/>

**Comment by PermeationLoop** [comment]
小太陽更新：畫好拉！

<img width="2423" height="1527" alt="小太陽背景" src="https://github.com/user-attachments/assets/de9178cc-ffb9-4aee-94a4-7a490d3f7dc4" />

歡迎來踩踩我的 QQ 空間🥰👍🏻
https://www.pixiv.net/artworks/123552105

**Comment by PermeationLoop** [comment]
參考 @Sayafx 提供的參考書 [👉🏻博イラスト集〜明日ちゃんまでの足跡〜](https://z-lib.gd/book/21745746/88f8da/) 畫的電子鉛筆畫

<img width="2294" height="2560" alt="模擬鉛筆畫" src="https://github.com/user-attachments/assets/59eaeff7-38f6-4e05-a556-432bf6bd4ebc" />

作者的形體真是一流，模仿一下都大大漲姿勢了，而且模擬鉛筆畫給我一種前所未有的電繪感覺🥰
